src/pages/login/registerClient.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <form>
+    <slot />
+    <input
+      type="text"
+      placeholder="nombre" />
+    <input
+      type="text"
+      placeholder="correo electrónico" />
+    <input
+      type="dni"
+      placeholder="DNI" />
+    <input
+      type="password"
+      placeholder="contraseña" />
+    <input
+      type="password"
+      placeholder="confirmar contraseña" />
+    <input
+      type="categoria"
+      placeholder="categoria" />
+    <Boton
+      type="auth"
+      @click="createAcc('/client/createEvent')">
+      {{ $t('Crear') }}
+    </Boton>
+    <p class="message">
+      {{ $t('tienesCuenta') }} <a href="/login">{{ $t('iniciaSesion') }}</a>
+    </p>
+  </form>
+</template>
+
+<route lang="yaml">
+  meta:
+    layout: login
+</route>
+
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
```

Comment:
```suggestion
import { useRouter } from 'vue-router/auto';
```

Actualiza también el resto de importaciones de los archivos de este PR porfa.

Tengo que forzar a nivel de linter esto, pero al importar el ``auto``, utilizamos la versión de Vue Router que tiene type checking y te autocompleta las rutas y los parámetros y todo con TypeScript.

Es un poco mierda el tener que acordarse de esto (por eso voy a intentar forzarlo a nivel de linter) pero el type checking de las rutas ahora mismo es experimental en la librería que utilizamos y esto es lo que hay 🤦‍♂️